packages/contracts/src/contracts/current/utils/Memory/Memory.sol

```diff
@@ -0,0 +1,86 @@
+/*
+
+  Copyright 2018 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.4.21;
+
+contract Memory {
```

Comment:
We can call it `LibMemory` to emphasize that it is just a collection of pure functions. (Like `LibOrder`)

Reply:
I agree. Wasn't sure on the convention here, since other utilities (like SafeMath) don't have the Lib prefix. @abandeali1 thoughts?

Reply:
I like the `Lib` prefix. Maybe `LibBytes` is a better name?